crates/bevy_render/src/camera/projection.rs

```diff
@@ -237,6 +240,16 @@ impl CameraProjection for OrthographicProjection {
                     (min_width, height * min_width / width)
                 }
             }
+            ScalingMode::AutoMax {
+                max_width,
+                max_height,
+            } => {
+                if width * max_height < max_width * height {
```

Comment:
Hey, can you provide some clarification on why this must be `width * max_height < max_width * height`?

Reply:
Yeah sure I’ll try to explain it. Will probably be done by tomorrow :)

Reply:
Nvm just done it. Think this is ok. Or is it still messy? I’m not really good at explaining something in few words. 😅